src/fontstash.h

```diff
@@ -781,6 +822,12 @@ void fonsSetFont(FONScontext* stash, int font)
 	fons__getState(stash)->font = font;
 }
 
+void fonsSetOversample(FONScontext* stash, int oversampleH, int oversampleV)
+{
+	fons__getState(stash)->oversampleH = oversampleH;
```

Comment:
I think this is my biggest question right now to move forward. It also can address FreeType issue. IF we decide to just have pixel-snapping on/off (off giving 2x2 oversampling), then pixel snapping off can just revert to sub-pixel rendering in FreeType (which works very well on its own). So we end up with sub-pixel, or pixel-snapping, and it looks good with both STBTT and FreeType. I really think this is good...